pkg/universe.dagger.io/python/python.cue

```diff
@@ -2,28 +2,60 @@
 package python
 
 import (
-	"universe.dagger.io/docker"
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
 
+	"universe.dagger.io/docker"
 	"universe.dagger.io/alpine"
 )
 
 // Run a python script in a container
 #Run: {
 	// Contents of the python script
-	script: string
+	script: {
+		// A directory containing one or more bash scripts
+		directory: dagger.#FS
+
+		// Name of the file to execute
+		filename: string
+
+		_directory: directory
+		_filename:  filename
+	} | {
+		// Script contents
+		contents: string
+
+		_filename: "run.py"
+		_write:    core.#WriteFile & {
+			input:      dagger.#Scratch
+			path:       _filename
+			"contents": contents
+		}
+		_directory: _write.output
+	}
+
+	// arguments to the script
+	args: [...string]
+
+	// where to mount the script inside the container
+	_mountpoint: "/run/python"
 
-	// FIXME: don't pass the script as argument: write to filesystme instead
 	docker.#Run & {
 		command: {
-			name: "python"
-			flags: "-c": script
+			name:   "python3"
```

Comment:
What if someone is using `python2`,  will it works?

Reply:
No it won't, but also `python2` isn't being installed in the default image and hasn't been supported since 2020. I don't think we need to take pains to support it.

If users need support for Python 2 I could allow `command` to be overridden by the caller like `args` is and users can make their own #Image to pass in? Is that sufficient?

Reply:
I think we can leave it as is, and if anyone complains about it, we can make it more extensible later.

Reply:
This should just be `python`.

Reply:
That actually fails. Alpine's python3 package does not symlink python to python3.

We could (and should! I may submit as another pr...) Use the python3-alpine image rather than building our own.